src/common/prop.c

```diff
@@ -319,9 +319,24 @@ daos_prop_valid(daos_prop_t *prop, bool pool, bool input)
 				prop->dpp_entries[i].dpe_str))
 				return false;
 			break;
-		/* container-only properties */
 		case DAOS_PROP_PO_SVC_LIST:
 			break;
+		case DAOS_PROP_PO_SCRUB_SCHED:
+			val = prop->dpp_entries[i].dpe_val;
+			if (val != DAOS_SCRUB_SCHED_OFF &&
```

Comment:
If these values are contiguous, could we do something like >= and <= to cut down on checks (with some static asserts to ensure they remain so)

Reply:
I agree that's better. I'll look into doing that.